Respondant - Acoustic Merc Ship Made by 100% True TSF (American) Patriots and Not USSP Engineers

grandalff opened the pull request:
## About the PR
Adds respondant, a nimble and fast merc ship, designed for shock combat and rescues. Works as either a fleet escort or an armored rescue ambulance.

## Why / Balance
A speedy, shock-style fighter capable of quickly responding to situations or threats (due to speed and radar) seems alot more unique than every ship being 10 big cannon shoot you from the same range.

## How to test
Spawning in as a merc, buying it on the civ shipyard.

## Media
note: may be changed slightly, as my file somehow got corrupted and no save loaded, except one on a new branch, so I re-did the latest changes.
<img width="671" height="789" alt="Screenshot 2025-08-28 213848" src="https://github.com/user-attachments/assets/2811dfb5-fd52-45df-813b-dd2bcb05e05b" />
<img width="525" height="599" alt="Screenshot 2025-08-28 213858" src="https://github.com/user-attachments/assets/df9502e5-bc07-472c-bd91-84c0fdaab510" />
<img width="1273" height="894" alt="Screenshot 2025-08-28 213937" src="https://github.com/user-attachments/assets/1aeb5fd1-db66-4a60-9329-ddfe42612638" />
<img width="1222" height="909" alt="Screenshot 2025-08-28 213957" src="https://github.com/user-attachments/assets/792f5116-7b1c-4aa7-8e38-534588e318f5" />
<img width="852" height="339" alt="Screenshot 2025-08-28 214018" src="https://github.com/user-attachments/assets/c6761720-a849-4d5b-aa46-71a9d054abbf" />
<img width="473" height="274" alt="Screenshot 2025-08-28 214024" src="https://github.com/user-attachments/assets/015aecfb-a329-4849-a472-5bdd73ebeac5" />
<img width="513" height="399" alt="Screenshot 2025-08-28 214030" src="https://github.com/user-attachments/assets/a4f9852b-6754-4a75-a461-6b83ebbe9a68" />
<img width="424" height="382" alt="Screenshot 2025-08-28 214041" src="https://github.com/user-attachments/assets/ae69f9db-246d-46d8-b742-b8372160f688" />
<img width="419" height="312" alt="Screenshot 2025-08-28 214048" src="https://github.com/user-attachments/assets/c6ae13ef-3c8f-415c-8a11-067ff9884e77" />
<img width="524" height="359" alt="Screenshot 2025-08-28 214056" src="https://github.com/user-attachments/assets/28279a1b-e635-4c41-bcf9-56a3484ad33d" />
<img width="564" height="427" alt="Screenshot 2025-08-28 214102" src="https://github.com/user-attachments/assets/e26214cb-dea4-4838-b2fb-f320811afcc5" />
<img width="498" height="558" alt="Screenshot 2025-08-28 214109" src="https://github.com/user-attachments/assets/aef54ec7-c4f8-416b-9685-08f75156a6ac" />
<img width="788" height="480" alt="Screenshot 2025-08-28 214124" src="https://github.com/user-attachments/assets/cf10995d-befa-4caa-b1ca-909a8e179090" />
<img width="816" height="516" alt="Screenshot 2025-08-28 214129" src="https://github.com/user-attachments/assets/fd26f411-746e-4bc7-bca2-4653ee4b4607" />
<img width="679" height="669" alt="Screenshot 2025-08-28 214211" src="https://github.com/user-attachments/assets/6113ce0d-f59c-4291-89f3-4454ff4f4036" />
<img width="137" height="175" alt="Screenshot 2025-08-28 215325" src="https://github.com/user-attachments/assets/628c878a-328e-43ea-95cb-c008b949cf67" />
<img width="944" height="757" alt="Screenshot 2025-08-28 235315" src="https://github.com/user-attachments/assets/534eed57-4cce-44de-afdd-7a36c9b23853" />
<img width="507" height="678" alt="Screenshot 2025-08-29 000300" src="https://github.com/user-attachments/assets/e1684873-ab99-469d-815c-e39ef2b98f1b" />

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
Please make sure it doesn't somehow break and not load. It worked fine for me now, but idk.

## Breaking changes

**Changelog**

:cl:
- add: Added Respondant, an armed and armored merc ambulance and/or fleet escort, definitely built by true TSF patriots and not USSP engineers wanting more pay via capitalism. Is meant for shock combat, and meant to crumble under sustained fighting. Aka be quick or be dead.

grandalff committed:
respondant

grandalff committed:
respondant da

grandalff committed:
IT WORKETH

grandalff committed:
testing if this fixes da test

monolith8319 committed:
chore: automatically update REUSE headers

grandalff commented:
I have no fucking idea why it is failing the new test. Tommorrows issues!!!

OneZerooo0 reviewed (changes requested):
Too much gun, both inside and outside - Torpedos? And a full gun rack? And armor? And two really really strong pistols?
Ontop of this being a medical ship… which is fine on its own, but this is such a heavily armed/threatening medship - that is available to civs? (Access locked but if it’s a merc ship it should go in the exped category.)
Tone it down a ton. Also, decals could use more work

grandalff commented:
Sellsword, claymore are merc ships, but not in the exped category.

OneZerooo0 commented:
I mean the exped shipyard

grandalff commented:
Theyre also not in the exped shipyard actually, I can move em there if needed ig

grandalff committed:
changes

grandalff committed:
change 2

grandalff commented:
Replaced all guns with two SVS, aka better mosin, but still outclassed by everything. Replaced torpedos with GPOB-L, not loaded, put 4 zenith's in armory. Removed one tarnyx shell from the rack, I guess. Removed the smart pistols. As for decals, I have no idea what to change without making the ship very noisy and bombastic.
<img width="977" height="699" alt="Screenshot 2025-08-31 004156" src="https://github.com/user-attachments/assets/825e6ded-1d90-4bf0-87dd-a5337d2a956b" />
<img width="701" height="355" alt="Screenshot 2025-08-31 004210" src="https://github.com/user-attachments/assets/73be668c-33b4-4c3d-9712-52bb8e644077" />
<img width="447" height="247" alt="Screenshot 2025-08-31 004215" src="https://github.com/user-attachments/assets/6796bc29-62d6-4638-9a73-456f50158c8d" />
<img width="747" height="751" alt="Screenshot 2025-08-31 004224" src="https://github.com/user-attachments/assets/6c6ba4d0-fd61-4b62-b9b2-6b0e82552ba4" />

monolith8319 committed:
chore: automatically update REUSE headers

grandalff commented:
<img width="634" height="398" alt="image" src="https://github.com/user-attachments/assets/22c8b357-6331-40ca-a2e2-29b78ccffd28" />
<img width="1232" height="829" alt="image" src="https://github.com/user-attachments/assets/84172e49-881f-48f3-845d-001a6360a748" />
Moved to exped console

OneZerooo0 reviewed (changes requested):
Just fix the test fails and it’s probably fine

grandalff commented:
The test was ran prior to these fixes, ima just move a plush abit and re-push it :godo:

grandalff committed:
I REMAIN ALONE

monolith8319 committed:
chore: automatically update REUSE headers

grandalff committed:
Merge remote-tracking branch 'upstream/main' into merc-ship--please-

grandalff committed:
uhh will this work

grandalff committed:
Revert "uhh will this work"

This reverts commit d7a327e03fb04527911c05fbb6cf5f8be6abd3f8.

grandalff committed:
me when I

grandalff commented:
WOOOOOOOOOOOOOOOOOO NO TEST FAILS

grandalff committed:
Merge remote-tracking branch 'upstream/main' into merc-ship--please-

grandalff committed:
uhh should add roofs?

monolith8319 committed:
chore: automatically update REUSE headers

OneZerooo0 reviewed (changes requested):
wait just a second
It starts with SCAF suits??

OneZerooo0 commented:
SCAF shouldn’t be roundstart on ships, the merc hardsuits are fine though

Avalon-Proto commented:
> SCAF shouldn’t be roundstart on ships, the merc hardsuits are fine though

Sacf suits are on at least 2-3 other ships. Like the Sirius

OneZerooo0 commented:
Doesn’t mean that they *should* be

grandalff commented:
Sirius is made by Starch, the head maintainer.

OneZerooo0 commented:
Still doesn’t mean that they should be, my statement still stands

grandalff committed:
removes the shitty suits, replaces with merc

grandalff commented:
updated

monolith8319 committed:
chore: automatically update REUSE headers

Skyguard12 commented:
Weird. Every merc in the known universe already starts with a scaf suit. Extra suits are just nice for spacers that ask you to take them on your ship.

OneZerooo0 reviewed (approved):
Approved after some maintainer discussion

grandalff commented:
shouldnt it be.... merged
